Makefile

```diff
@@ -35,6 +35,9 @@ mypy:
 pylama:
 	pipenv run pylama --options setup.cfg
 
+pydocstyle:
+	pipenv run pydocstyle
```

Comment:
We probably only want to run pydocstyle on the sapinvoices package (not the tests), so this would be `pipenv run pydocstyle sapinvoices`.